pandas/core/generic.py

```diff
@@ -1831,11 +1838,43 @@ def to_hdf(self, path_or_buf, key, **kwargs):
             Specifying a compression library which is not available issues
             a ValueError.
         fletcher32 : bool, default False
-            If applying compression use the fletcher32 checksum
-        dropna : boolean, default False.
+            If applying compression use the fletcher32 checksum.
+        dropna : bool, default False
             If true, ALL nan rows will not be written to store.
-        """
 
+        See Also
+        --------
+        DataFrame.read_hdf : read from HDF file.
+        DataFrame.to_parquet : write a DataFrame to the binary parquet format.
+        DataFrame.to_sql : write to a sql table.
+        DataFrame.to_feather : write out feather-format for DataFrames.
+        DataFrame.to_csv : write out to a csv file.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]},
+        ...                   index=['a', 'b', 'c'])
+        >>> df.to_hdf('data.h5', key='df', mode='w')
+
+        We can append another object to the same file:
+
+        >>> s = pd.Series([1, 2, 3, 4])
+        >>> s.to_hdf('data.h5', key='s')
+
+        Reading from HDF file:
+
+        >>> pd.read_hdf('data.h5', 'df')
+        A  B
+        a  1  4
+        b  2  5
+        c  3  6
+        >>> pd.read_hdf('data.h5', 's')
+        0    1
+        1    2
+        2    3
+        3    4
+        dtype: int64
+        """
```

Comment:
Can you add in the end here a code block with

```
>>> import os
>>> os.remove('data.h5')
```

(so running the doctests does not leave behind files)

Reply:
done

Many thanks for you comments, they are really useful.